lib/ws-kernel-picker.coffee

```diff
@@ -0,0 +1,117 @@
+{SelectListView} = require 'atom-space-pen-views'
+_ = require 'lodash'
+
+Config = require './config'
+services = require './jupyter-js-services-shim'
+WSKernel = require './ws-kernel'
+uuid = require 'uuid'
+
+class CustomListView extends SelectListView
+    initialize: (@emptyMessage, @onConfirmed) ->
+        super
+        @addClass('watch-language-picker')
+        @storeFocusedElement()
```

Comment:
I had a quick look into it:
The problem is the `CustomListView`s created in `onGateway` and `onSession` try to restore the focus to the input field of `CustomListView`.

I have an easy fix for that and submit a PR after this is merged.